pkg/operator/controller/garden/garden/components.go

```diff
@@ -389,7 +389,7 @@ func (r *Reconciler) newVirtualGardenGardenerResourceManager(secretsManager secr
 		false,
 		nil,
 		true,
-		[]string{v1beta1constants.GardenNamespace, metav1.NamespaceSystem, gardencorev1beta1.GardenerShootIssuerNamespace},
+		[]string{v1beta1constants.GardenNamespace, metav1.NamespaceSystem, gardencorev1beta1.GardenerShootIssuerNamespace, gardencorev1beta1.GardenerSystemPublicNamespace},
```

Comment:
Why is this required? As far as I understand the change, `gardener-apiserver` will create/update the `ConfigMap`. There are no `ManagedResource`s involved. Why does `gardener-resource-manager` need this namespace?

Reply:
GRM is create the role and rolebinding allowing `system:authenticated` group to access the configmap.